api/assessments/routes.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+from api.responses import assessment_201_response
+from api.responses import error_response
+from db.models.assessment import AssessmentError
+from db.models.assessment import AssessmentMethods
+from flask import request
+from flask import Response
+from flask.views import MethodView
+
+
+class AssessmentsView(AssessmentMethods, MethodView):
+    def search(self):
+        """
+        GET /assessments endpoint
+        :return: Json Response
+        """
+        return Response(
+            json.dumps(self.assessments(as_json=True)),
+            mimetype="application/json",
+        )
```

Comment:
💯 Nice to have this wrapped up in the response object...we should all try to use this...

Reply:
👍